lib/sqlalchemy/orm/util.py

```diff
@@ -524,6 +524,17 @@ def _reconstitute_from_aliased_insp(cls, aliased_insp):
         obj = cls.__new__(cls)
         obj.__name__ = "AliasedClass_%s" % aliased_insp.mapper.class_.__name__
         obj._aliased_insp = aliased_insp
+        for poly in aliased_insp.with_polymorphic_mappers:
```

Comment:
this is duplicating code that already exists in AliasedInsp, it should be in a method.    I want to get this in so I can release so I'll do it.

Reply:
Thank you for taking it on.